src/engine/MaxConstraint.h

```diff
@@ -262,10 +246,28 @@ class MaxConstraint : public PiecewiseLinearConstraint
                  : static_cast<unsigned>( phase ) - MAX_VARIABLE_TO_PHASE_OFFSET;
     }
 
+    /*
+      Eliminate the case corresponding to the given input variable to Max.
+    */
+    void eliminateCase( unsigned variable );
+
     /*
       Return true iff f or the elements are not all within bounds.
     */
     bool haveOutOfBoundVariables() const;
+
+    /*
+     * Returns a boolean value indicating if at least one input variable was eliminated (True)
+     * or not (False)
+     */
+    bool wereVariablesEliminated() const;
```

Comment:
Maybe hasEliminatedVariables()? In terms of OO design, the question wereVariablesEliminated() implicitly involves time, which is unnecessary.

Reply:
Removed this method since it's no longer used anywhere.